chapters/3-measurement.tex

```diff
@@ -37,125 +39,209 @@
+While the time to create the data plan is before any data has been acquired,
+it should be a piece of documentation that is 
+continuously updated as the project evolves.
+
+To create a data plan according to DIME's template, first,
+start by creating a \textbf{data linkage table} by listing 
+all the datasets you know you will use in a spreadsheet.
+If one source of data will result in two different datasets, 
+then list each dataset on its own row.
+For each dataset, list the \textbf{unit of observation}\sidenote{
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}},
+and the name of the project ID variable for that unit of observation.
+Your project should only have one project ID variable per unit of observation. 
+When you list a dataset in the data linkage table --
+which should be done before that dataset is acquired --
+you should always make sure that the dataset will
+be fully and uniquely identified by the project ID, 
+or make a plan for how 
+the new dataset will be linked to the project ID.
+It is very labor intensive to work with a dataset that
+do not have an unambiguous way to link to the project ID, 
+and it is a major source of error.
+
+The data linkage table should indicate whether 
+datasets should be merge one-to-one, for example, 
+merging baseline data and endline data that use the same unit of observation,
 or whether two datasets should be merged many-to-one, 
 for example, school administrative data merged with student data.
-Your data map must indicate which ID variables can be used and how to merge datasets.
-It is common that administrative data use IDs 
-that are different than the project IDs, 
-and the linkage between those should be clear from your master dataset.
-
-The data map should also include metadata about the handling of all information.
-These characteristics may be updated as the project progresses.
-For example, you will need to note the original source of each dataset,
-as well as the project folder where 
-the raw original data and codebooks are stored
-and where the back-ups for the each raw dataset are stored.
-
-Some of the characteristics in your master datasets and your data map
-should be filled in during the planning stage,
-but both of them should be active resources 
-that are updated as your project evolves.
-Finally, your master data should not include any unlabeled missing values. 
-If the information is missing for one unit, 
-then the reason should always be indicated with a code.
-An example for such reason could be that a unit was not included in the treatment assignment
-as it was not sampled in the first place,
-or was not located in the data collection at a given round.
-
-\subsection{Defining study comparisons using master data}
-
-Your research design will determine what statistical comparisons you need
-to estimate in your analytical work.
+Your data map must indicate which ID variables 
```

Comment:
i think we need a clear definition, earlier on in this chapter, of project ID specifically and ID variables generally.

Reply:
done in dd5fb8ae2207f5cd9d9a8baa43a634ae2edd985a and 9182ca3a07adf589000e2231349fb7c379dfb6fe